web/components/editor.tsx

```diff
@@ -56,6 +63,71 @@ export function useTextEditor(props: {
           class: clsx('no-underline !text-indigo-700', linkClass),
         },
       }),
+      Mention.configure({
+        HTMLAttributes: {
+          class: clsx('not-prose text-indigo-700', linkClass),
+        },
+        // TODO: do a Next link instead of raw <a>
+        renderLabel: ({ options, node }) =>
+          [
+            'a',
+            { href: node.attrs.label },
+            `${options.suggestion.char}${node.attrs.label ?? node.attrs.id}`,
+          ] as any,
+        suggestion: {
+          items: ({ query }) =>
+            users
```

Comment:
Should these be sorted? E.g. alphabetically by name.

I think you'll also find that names with a prefix match should come before those with a mid-string match.

Reply:
This logic could be defined in a named function at the top level of this file (or possibly a util file).

Reply:
> Should these be sorted?

The best UI would be sorting by your follows > your followers > number of followers.
The 80/20 UI is sorting by account balance I guess?

Reply:
Yup. Something pretty important is to show the prefix matches first. So, if I search for "Si" it should put Sinclair above names that merely contain "si", like Farsi (or whatever).